docs/Demo.md

```diff
@@ -39,19 +39,18 @@ The second argument the script takes is the url of the forked repo, so for examp
 An "idle" state could resemble:
 ![Idle-Data](img/pelorus-dashboard-idle-data.png)
 
-    >:mag: **Note**<br/>
```

Comment:
I read through the mkdocs and python-markdown docs and couldn't find what `:mag:` was supposed to be. I can add it back if I can be shown how it works!

Reply:
It's this... :mag: its part of the github flavored markdown emoji set. Looks like none of this callout stuff renders properly in mkdocs... let me see if I can find an alternative, since this whole section looks like trash in readthedocs.

Reply:
bummer.... https://github.com/readthedocs/readthedocs.org/issues/3098

Reply:
Oh, we could just embed the emoji themselves, then. It's Just Text™!